.github/workflows/release.yml

```diff
@@ -98,6 +99,18 @@ jobs:
           echo '::set-env name=target-directory::.build/magento1'
           echo '::set-env name=build-command::make magento1-build'
 
+      - name: Prepare release for MagentoV2
+        if: startsWith(github.ref, 'refs/tags/MagentoV2-')
+        run: |
+          PACKAGE_VERSION=${{ steps.packageVersion.outputs.tag }}
+          echo '::set-env name=extensions::curl, json, mbstring, xdebug'
+          echo "::set-env name=release-name::magento_v1-${PACKAGE_VERSION//\./_}"
+          echo '::set-env name=php-versions::7.3'
+          echo '::set-env name=key::cache-magentov2-v2'
+          echo '::set-env name=source-directory::magento2'
+          echo '::set-env name=target-directory::.build/magento2'
+          echo '::set-env name=build-command::make magento2-build'
```

Comment:
I don't think you have the makefile yet

Reply:
how to build the file ?

Reply:
See `makefile`, `build` & `build-opencartv1`
Can also see https://github.com/indodana/paylater-ecommerce-plugin-php/pull/17/ for example for Prestashop case

Reply:
done